app/views/RoomMembersView/index.js

```diff
@@ -163,24 +168,108 @@ class RoomMembersView extends React.Component {
 		}
 	}
 
+	handleRemoveFromTeam = async(selectedUser) => {
+		const { navigation } = this.props;
+		const { room } = this.state;
+		const db = database.active;
+		const subCollection = db.get('subscriptions');
+		const teamChannels = await subCollection.query(
+			Q.where('team_id', room.teamId),
+			Q.where('team_main', null)
+		);
+		if (teamChannels) {
+			navigation.navigate('SelectListView', {
+				title: 'Remove_Member',
+				infoText: 'Remove_User_Teams',
+				data: teamChannels,
+				nextAction: (selectedChannels = {}) => this.removeFromTeam({ selectedUser, selectedChannels })
+			});
+		} else {
+			showConfirmationAlert({
+				message: I18n.t('Removing_user_from_this_Team', { user: selectedUser.username }),
+				confirmationText: I18n.t('Yes_action_it', { action: I18n.t('remove') }),
+				onPress: () => this.removeFromTeam({ selectedUser })
+			});
+		}
+	}
+
+	removeFromTeam = async(params) => {
```

Comment:
You're destructing the params on your call, but using naming as `params` here.
This way is harder to find bugs on your code.
If your code was like this, you could have found the bug earlier:
```js
removeFromTeam = async({ selectedUser, selected }) => {
```